src/components/Tabs.tsx

```diff
@@ -97,9 +92,9 @@ class Tabs extends React.Component<any, any> {
 
     render() {
         // ensures that single or no child components don't throw
-        const children = /** @type {React.Component<TabProps>[]} */ React.Children.toArray(
+        const children = React.Children.toArray(
             this.props.children
-        )
+        ) as React.Component<TabProps>[]
```

Comment:
I don't think there's any way to extract the React Component type from the JSX Element type, or even to make the JSX Element more specific, so a casting is probably required here.

The closest I could come with googling was this issue: https://github.com/microsoft/TypeScript/issues/21699 which is still WIP.

Reply:
^ cc @proxi Please let me know if you can find me a better alternative here. 🙏

Reply:
I think this is fine, there's no good solution.